docs/releases/3.0.txt

```diff
@@ -54,6 +54,11 @@ This is in addition to our existing WSGI support. Django intends to support
 both for the foreseeable future. Async features will only be available to
 applications that run under ASGI, however.
 
+Async support only applies to the outer ASGI application. Internally everything
```

Comment:
I think I'd just begin this sentence `At this stage async support...`

Reply:
Updated